hw/hw06/series.py

```diff
@@ -0,0 +1,80 @@
+# HW6 Mathmatical Series
+
+
+def fibonacci(n):
+    """Defining a function  called fibonacci to provide nth value in fibonacci
+    sequence."""
+    if n == 0:
+        return 0
+    # First value in fibonacci sequence is 0.
+    elif n == 1:
+        return 1
+    # Second value in fibonacci sequence is 1
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+    """Formula for fibonacci sequence after first two values.  Function calls
+    itself and sums the values to provide the nth value starting from 0."""
+
+
+def fib(n):
+    """Function to print fibonacci function"""
+    print (fibonacci(n))
+
+
+fib(3)  # test to see if fibonacci function is printing correct value
+fib(5)  # test to see if fibonacci function is printing correct value
+
```

Comment:
You could also just call your function with a print for a test instead of writing a function that only prints.

```print(fibonacci(3))  # Expected = 2```